_scripts/committee.sh

```diff
@@ -9,6 +9,13 @@
 #
 
 committee=$1
+
+if [[ -z "$committee" ]]; then
+	echo "Missing argument"
+	echo "Usage: $0 [BASE CMT IMAGES DIR]"
+	exit 1
+fi
```

Comment:
Not that big of a deal at all but just as a note for the future this change should probably be it's own separate PR.